src/components/20-molecules/accordion/index.js

```diff
@@ -35,6 +35,7 @@ class AXAAccordeon extends LitElement {
   constructor() {
     super();
     applyDefaults(this);
+    this.debouncedHandleViewportCheck = debounce(this.toggleAnimation, 500);
```

Comment:
Requires a comment saying what problem is solved here by debouncing and explaining the magical 500 milliseconds!

Reply:
this.debouncedHandleViewportCheck is not a great name. That it's debounced can be inspected locally at the method, it's a minor detail.

How about this.recalculateDimensionsOnResize?